openedx/core/djangoapps/programs/utils.py

```diff
@@ -136,19 +138,31 @@ def progress(self, programs=None, count_only=True):
                 progress, and unstarted courses instead of serialized representations
                 of the courses.
 
+            show_expired_runs_as_remaining (bool): Whether in progress runs whose upgrade
+                deadline has expired should be counted as remaining rather than in progress.
+
         Returns:
             list of dict, each containing information about a user's progress
                 towards completing a program.
         """
+        now = datetime.datetime.now(tzlocal())
```

Comment:
Is passing `tzlocal()` required? Django should take care of timezone detection for you.